frontend/catalyst/pennylane_extensions.py

```diff
@@ -206,7 +216,7 @@ def _make_jaxpr_check_differentiable(f: Differentiable, grad_params: GradParams,
 def _check_grad_params(
     method: str, h: Optional[float], argnum: Optional[Union[int, List[int]]]
 ) -> GradParams:
-    methods = {"fd", "ps", "adj"}
+    methods = {"fd", "mixed"}
```

Comment:
Not a blocking comment for this PR :) But I'm wondering if there is a word we can use here (rather than `mix`) that indicates `catalyst.grad` will simply defer to the classical/quantum components for the specified diff method 🤔

Reply:
I can't think of anything myself

Reply:
Per today's standup, I updated the API to use `"defer"`.